tokens/mainnet.json

```diff
@@ -1,4 +1,5 @@
 {
   "networkId": 0,
-  "tokens": []
+  "tokens": [],
+  "nftCollections": []
```

Comment:
do we really want everything in one file? in my [previous proposition ](https://github.com/alephium/token-list/pull/9) I rather introduced
```
nfts/mainnet,json
```

Reply:
Good question, i thought about it and was a bit unsure as well
1. This repo is called `token-list`, which should be a list of tokens. Would be a little bit weird if we have `TokenList` as well as a separate `NFTList`. 
2. NFTs are tokens as well, just non-fungible. So it feels that `NFTList` is conceptually a subset of `TokenList`.  That is why I am adding a `nftCollections` field in the `TokenList`.

WDYT? I don't have a very strong opinion on this, so I am ok to have them in separate files too.

Reply:
I don't have strong opinion either :sweat_smile:  I guess it depends how we will use it. Maybe we can ask @nop33 expertise on this?

Reply:
Fungible tokens and non-fungible tokens are both tokens, so it's not weird to see them both in a token-list project. Now the question is: Do we include the NFT collections inside the `tokens/[mainnet|testnet].json` or do we create a new `nfts/[mainnet|testnet].json`? I think both are fine. I am slightly leaning toward the latter because:
1. It'll make it easier to add new tokens and nft collections. If they are in 1 file one will need to scroll through a long file to figure our where can they add a new collection.
2. Seeing both `tokens` and `nfts` from the root of the project makes it easier to navigate it

Both of these points are really not important, especially for the beginning. Just my thoughts.

Reply:
I will create a separate file for NFT, thanks for the input guys! :+1:

Reply:
There is also a performance benefit to use two files. If only one file is used in a project, then Webpack could potentially remove the unused JSON from the bundle.